cmake/private/FairRootSummary.cmake

```diff
@@ -0,0 +1,218 @@
+################################################################################
+# Copyright (C) 2014-2021 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH  #
+#                                                                              #
+#              This software is distributed under the terms of the             #
+#              GNU Lesser General Public Licence (LGPL) version 3,             #
+#                  copied verbatim in the file "LICENSE"                       #
+################################################################################
+
+function(fairroot_summary_header)
+  message(STATUS "  ")
+  message(STATUS "               C O N F I G U R A T I O N   S U M M A R Y")
+  message(STATUS "  ")
+  message(STATUS "  ${Cyan}PROJECT NAME${CR}         ${BGreen}${PROJECT_NAME}${CR}")
+  message(STATUS "  ${Cyan}VERSION${CR}              ${BGreen}${PROJECT_VERSION}${CR}")
```

Comment:
Would be cool, if we print also the git version in brackets, if available and different from `PROJECT_VERSION`. But doesn't need to happen in this PR.

Reply:
Added FAIRROOT_GIT_VERSION in brackets, if it is set.